examples/ticks_and_spines/spines_demo.py

```diff
@@ -3,10 +3,10 @@
 Spines
 ======
 
-This demo compares :
-    - normal axes, with spines on all four sides
-    - an axes with spines only on the left and bottom.
-    - an axes using custom bounds to limit the extent of the spine
+This demo compares:
+    - normal axes, with spines on all four sides;
+    - an axes with spines only on the left and bottom;
+    - an axes using custom bounds to limit the extent of the spine;
```

Comment:
FYI, I didn't even bother doing it on my computer: I just did it through the github API.